orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -119,6 +132,9 @@ def __init__(self, module):
         self._save_onnx = False
         self._save_onnx_prefix = ''
 
+        # CPP extension to get torch CUDA allocator's alloc and free function addresses
+        self._torch_cuda_allocator = load_inline(name='inline_extension', cpp_sources=[torch_cuda_allocator_addresses_cpp_source], functions=['cuda_caching_allocator_raw_alloc_address', 'cuda_caching_allocator_raw_delete_address'], verbose=True, with_cuda=True)
```

Comment:
Given the previous comment, this line would become something like
`self._torch_cuda_allocator = _load_external_cuda_allocator(...)`